appraisal.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
+  s.required_ruby_version = ">= 2.3.0"
```

Comment:
This change in particular will make the gem unusable for earlier Ruby versions, rather than just unsupported. The app code doesn't use anything that requires Ruby 2.3; only the spec code does.

Reply:
Yes, that's intended.